manticore/wasm/types.py

```diff
@@ -0,0 +1,226 @@
+import typing
+from dataclasses import dataclass
+from ..core.smtlib import issymbolic, BitVec
+import wasm
+
+# TODO - These need to be symbolic fixed-size representations
+U32: type = type("U32", (int,), {})
+U64: type = type("U64", (int,), {})
+Byte: type = type("Byte", (int,), {})
+
+
+class I32(int):
```

Comment:
Is there any good reason for those to be classes instead of functions?

Also, we use `classmethod` here without using the `cls` argument so if we keep it as class, it could be `staticmethod`s.

Reply:
Yes, the reason here being that the types of the values are used frequently for validation